src/Composer/Installers/Installer.php

```diff
@@ -198,4 +222,44 @@ private function getIO()
     {
         return $this->io;
     }
+
+    /**
+     * Look for installers set to be disabled in composer's extra config and
+     * remove them from the list of supported installers.
+     * Uses true, "all", and "*" as global values to disable all installers.
+     *
+     * @return void
+     */
+    protected function removeDisabledInstallers()
+    {
+        $extra = $this->composer->getPackage()->getExtra();
+
+        if (!isset($extra['installer-disable'])) {
```

Comment:
It may be worth to consider options `[]` and `false`?

Reply:
Given that `false` is the equivalent of not adding this option, I hadn't considered it before. Now that you bring it up I can see where it would be useful in a situation where [`wikimedia/composer-merge-plugin`](https://packagist.org/packages/wikimedia/composer-merge-plugin) is being used. I'll add it as an option. 

Is your thought that this would be a global, "all installers are enabled"?

Regarding an options array, what did you have in mind? I'm not at all opposed to having available options, but I'm not sure what options to include or what they would do? Or are you suggesting to have it available for future ideas?

Reply:
@niksamokhvalov 
Just wondering if you had a chance to review the latest commit updating this code. I've added `false` as an option, but am unsure what `options[]` you had in mind. I'm open to the idea, but am just unsure of what or how to implement.